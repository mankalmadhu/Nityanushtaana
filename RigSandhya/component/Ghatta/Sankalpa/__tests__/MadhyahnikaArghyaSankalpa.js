import React from 'react';
import ಮಾಧ್ಯಾಹ್ನಿಕಅರ್ಘ್ಯಸ೦ಕಲ್ಪ from '../MadhyahnikaArghyaSankalpa';
import Renderer from 'react-test-renderer';

jest.mock('../../Gayatri/GayatriMantra', () => 'ಗಾಯತ್ರೀಮ೦ತ್ರ' );
jest.mock('../../Gayatri/Vyahruti', () => 'ವ್ಯಾಹೃತಿ' );
jest.mock('../SankalpaPurvanga', () => 'ಸ೦ಕಲ್ಪಪೂರ್ವಾ೦ಗ' );
jest.mock('../SankalpaShershike', () => 'ಸ೦ಕಲ್ಪಶೀರ್ಷಿಕೆ' );
jest.mock('../Preetyartha', () => 'ಪ್ರೀತ್ಯರ್ಥ' );
jest.mock('../../../Vishaya/Sheershike',()=>'ಶೀರ್ಷಿಕೆ');
jest.mock('../Oomapo', () => 'ಓಮಾಪೋ' );

jest.mock('../../../Vishaya/Vishaya',()=>'ವಿಷಯ');

test("ಮಾಧ್ಯಾಹ್ನಿಕಅರ್ಘ್ಯಸ೦ಕಲ್ಪ", () => {

  expect(Renderer.create(
    <ಮಾಧ್ಯಾಹ್ನಿಕಅರ್ಘ್ಯಸ೦ಕಲ್ಪ />
  ).toJSON()).toMatchSnapshot();

});