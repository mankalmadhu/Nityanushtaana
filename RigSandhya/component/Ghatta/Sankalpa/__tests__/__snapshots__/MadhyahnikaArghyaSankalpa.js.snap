// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ಮಾಧ್ಯಾಹ್ನಿಕಅರ್ಘ್ಯಸ೦ಕಲ್ಪ 1`] = `
<View>
  <ಶೀರ್ಷಿಕೆ>
    ಅರ್ಘ್ಯ ಸ೦ಕಲ್ಪ
  </ಶೀರ್ಷಿಕೆ>
  <ಸ೦ಕಲ್ಪಪೂರ್ವಾ೦ಗ />
  <ವ್ಯಾಹೃತಿ />
  <ಗಾಯತ್ರೀಮ೦ತ್ರ />
  <ಓಮಾಪೋ />
  <ಪ್ರೀತ್ಯರ್ಥ>
    ಮಾಧ್ಯಾಹ್ನಿಕ ಸ೦ಧ್ಯಾ೦ಗ ಅರ್ಘ್ಯ ಪ್ರಧಾನಾನಿ ಕರಿಷ್ಯೇ ॥
  </ಪ್ರೀತ್ಯರ್ಥ>
</View>
`;
